test/unit/collector/remote-method.test.js

```diff
@@ -725,3 +728,68 @@ tap.test('when generating the User-Agent string', (t) => {
     t.end()
   })
 })
+
+tap.test('record data usage supportability metrics', (t) => {
+  t.autoend()
+
+  let endpoint
+
+  let agent
+
+  t.beforeEach(() => {
+    agent = helper.instrumentMockedAgent()
+    endpoint = {
+      host: agent.config.host,
+      port: agent.config.port
+    }
+  })
+
+  t.afterEach(() => {
+    agent && helper.unloadAgent(agent)
+  })
+
+  t.test('should aggregate bytes of uploaded payloads', async (t) => {
```

Comment:
I'm not sure if we have to be exhaustive with every endpoint but a few tests I could see valuable are:
 * asserting response byte size(you'll have to probably stub the response payload)
 * having the remote method return an error and still verify metrics are recording(current bug in PR)

Reply:
I've added several tests, including the ones you outlined here. Thanks for suggesting them!